redbot/cogs/mod/kickban.py

```diff
@@ -213,11 +225,24 @@ class KickBanMixin(MixinMeta):
 
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
+        author = ctx.author
+        guild = ctx.guild
+        if reason == None:
+            reason = "No reason was given."
+        self == self.bot.get_cog("Mod")
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle == True:
```

Comment:
`True` is singleton so you should either use identity check or since it's a bool, you can just use `if toggle:`
```suggestion
        if toggle:
```